common/cache/src/main/java/org/thingsboard/mqtt/broker/cache/TBRedisCacheConfiguration.java

```diff
@@ -122,7 +120,8 @@ public CacheManager cacheManager(RedisConnectionFactory cf) {
             }
         }
 
-        var redisCacheManagerBuilder = RedisCacheManager.builder(cf).cacheDefaults(configuration).withInitialCacheConfigurations(cacheConfigurations).transactionAware();
+        var redisCacheManagerBuilder = RedisCacheManager.builder(cf).cacheDefaults(configuration)
+                .withInitialCacheConfigurations(cacheConfigurations).transactionAware().disableCreateOnMissingCache();
```

Comment:
what is the purpose of adding the following method?
```
disableCreateOnMissingCache()
```

Reply:
To pass the test in the TbRedisCacheConfigurationTest that verifies that non-existing caches in specs cannot be created. Test:

```
@Test
public void givenCacheConfig_whenCacheManagerReady_thenVerifyNonExistedCaches() {
    assertThat(cacheManager.getCache("rainbows_and_unicorns")).isNull();
}
```

without `disableCreateOnMissingCache()` cacheManager.getCache("some_cache") will create a new instance of TransactionAwareCacheDecorator.

Reply:
good, approve